spec/analytics_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'spec_helper'
+
+RSpec.describe Analytics do
+
+  describe '#most_popular_make_model' do
+    it 'return the most popular make and model from EV registration' do
+      analytics = Analytics.new
+      ev_registrations = [ {:make=>"Tesla", :model=>"3"},
```

Comment:
Where would this data be coming from though? 
I would think maybe we'd send in a bunch of Vehicle objects from a Facility, and then the method in the class would parse through that data. Unless you have another method that formats data like this. 
We need to think about where the data for the arguments is coming from. Usually all we have are the objects already made from previous classes and the methods already written.